Data-Collection-Inventory/2021Q2_csvs/data_collection_specimen_type_classification.csv

```diff
@@ -0,0 +1,120 @@
+data_collection_reference_id,specimen_type_name
+dong_antero,brain section set
+dong_retro,brain section set
+dong_trio,brain section set
+dulac_hyp_sn_10xv2,cell nucleus
+ecker_hu_DNAseq,tissue sample
+ecker_hu_sn_ATACseq,cell nucleus
+ecker_hu_sn_mCseq,cell nucleus
+ecker_sn_ATACseq,cell nucleus
+ecker_sn_mCseq,cell nucleus
+ecker_sn_mCseq_retro,cell nucleus
+ecker_trio,brain section set
+feng_fish,cell in slice
+feng_fish,brain section set
+feng_morph,whole cell
+feng_sn_10xv3,cell nucleus
+feng_sn_dropseq,cell nucleus
+fischl_lsfm,tissue sample
+fischl_mri,brain hemisphere
+fischl_oct,brain slice
+fischl_stereology,tissue sample
+huang_antero,brain section set
+huang_fmost,whole brain
+huang_fmost_cells,whole cell
+huang_input_conn,whole brain
+huang_pn_bulk_SSv4,bulk nucleus
+huang_pn_sn_10xv2,cell nucleus
+huang_pn_sn_10xv3,cell nucleus
+huang_retro,brain section set
+huang_sn_10xv2,cell nucleus
+huang_sn_10xv3,cell nucleus
+huang_stpt_conn,whole brain
+huang_stpt_count,whole brain
+HumanM110x,cell nucleus
+human_cortex_SS4_counts,cell nucleus
```

Comment:
technically not biccn. Only the human_cortex_SS4_Open_GRU is BICCN; both restricted ones and the counts data collections are under Allen TOU; we can leave here for now, but may make sense to remove later.

Reply:
The parent project for this is currently `human_cortex`, which is part of `U01 Lein`. It sounds like we need to split this project into BICCN/non-BICCN parts because subprograms / grants are associated with a project, not a data collection. There is no way to know that this collection is not a part of `U01 Lein` based on looking at the raw data here.